errata_tool/erratum.py

```diff
@@ -948,7 +948,8 @@ def commit(self):
             # Errata tool is very slow - don't PUT if it hasn't changed
             allbugs = list(set(self.errata_bugs) | set(
                 self._cve_bugs) | set(self.jira_issues))
-            if sorted(self._original_bugs + self._original_jira_issues) != sorted(allbugs) \
+            originalbugs = self._original_bugs + self._original_jira_issues
```

Comment:
Might you want to use set logic to combine the original bugs and original jira issues.

We should also add some unittests around this logic since there is an error and I would not say that this would always do the right thing.